modules/core/src/Concept.js

```diff
@@ -76,7 +76,7 @@ export default class Concept {
       }),
     )
 
-    return files.flat().map((file) => file.replace(directory, '').replace(/^\//, ''))
+    return files.flat().map((file) => file.replace(directory, '').replace(/^\//, '').replace('\\', ''))
```

Comment:
This looks dangerous. We want to remove only leading slash. Better would be to replace all backslashes with regular slashes and then, remove the leading slash like is in the replace before. So something like this:

```
return files.flat().map((file) => file.replace(directory, '').replace(/\\/g, '/').replace(/^\//, '')
```